ansible/tasks/profile/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: Configure PAM to create $HOME if it doesn't exist
+  blockinfile:
+    path: /etc/pam.d/common-session
+    marker: "# {mark} ANSIBLE - create home if nonexistent"
+    block: |
+      session optional		pam_mkhomedir.so
```

Comment:
lineinfile rather than blockinfile, just match over a `pam_mkhomedir` because no doubt this will exist on some systems and might not exist on others

Reply:
also make sure that it matches `regexp:` with that whitespace thingy